examples/OcclusionSaliency.ipynb

```diff
@@ -26,12 +28,9 @@
     "- torch (tested with 1.9.0)\n",
     "- torchvision  (tested with 0.10.0)\n",
     "\n",
-    "### Miscellaneous\n",
-    "*Note*: When using the `from_config_dict` method below,\n",
-    "it is not unusual for the below code to throwback a warning\n",
-    "indicating missing psycopg2 and caffe modules.\n",
-    "Please disregard any such warnings.\n",
+    "These may be installed manually or via installing with the \"examples\" extra, e.g. ``poetry install -E example_deps``.\n",
     "\n",
+    "### Miscellaneous\n",
     "Reference for implementation: Zeiler MD, Fergus R. Visualizing \n",
```

Comment:
We may also want to consider citing RISE here as well.

Petsiuk V, Das A, Saenko K. Rise: Randomized input sampling for explanation of black-box models. arXiv preprint arXiv:1806.07421. 2018 Jun 19.

Reply:
I reorganized the "Misc" section a little to include a "References" heading.